src/features/Conversation/Error/InvalidAccess.tsx

```diff
@@ -35,6 +38,7 @@ const InvalidAccess: RenderErrorMessage['Render'] = memo(({ id }) => {
             value: Tab.Password,
           },
           { icon: <Icon icon={KeySquare} />, label: 'OpenAI API Key', value: Tab.Api },
+          { icon: <Icon icon={ScanFace} />, label: t('oauth', { ns: 'common' }), value: Tab.Oauth },
```

Comment:
I think this tab can't be display by default. This tab should only be enabled with an ENV like `ENABLE_OAUTH` or something like that.